apps/health_campaign_field_worker_app/lib/models/entities/user.dart

```diff
@@ -63,7 +64,7 @@ class UserModel extends EntityModel {
   final int? createdBy;
   final String? lastModifiedBy;
   final String? tenantId;
-  final String? uuid;
+  final List<String>? uuid;
```

Comment:
Is it supposed to be a list??

Reply:
@vishal-egov  for user search we need to pass it as an array hence we have made it as list  (profile screen)